### PR TITLE
Updated project structure to better match our current standards

### PR DESCRIPTION
**Pull request by pkruithof**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,35 +1,48 @@
...
+  "require-dev": {
+    "doctrine/orm": "^2.3",
+    "friendsofsymfony/user-bundle": "~2.0@dev",
+    "phpunit/phpunit": "^4.1",
+    "behat/behat": "^3.0",
+    "behat/symfony2-extension": "^2.0"
+  },
+  "suggest": {
+    "treehouselabs/keystone-bundle": "For API authentication that is easy to set up",
+    "behat/behat": "To test the API using Behat",
```
Comment (cleentfaar) [inline composer.json]: Behat dependancies in `require-dev` *en* in `suggest`?

**Comment by cleentfaar** [comment]
rest lgtm

File: composer.json (hunk 1 above)
Comment (pkruithof) [inline composer.json]: Yeah I thought a bit about this: they are not required for using the bundle (hence the suggests), except when you use the Behat contexts. But if we want to test the contexts we would need them as dev dependencies.

It seemed more appropriate than putting them all in `require`: which would create dependencies to eg. Behat and FOSUserBundle, even if you're not using the Behat tests.

Comment (cleentfaar) [inline composer.json]: :+1:

**Pull request merged**